packages/@aws-cdk/aws-apigateway/test/authorizers/integ.request-authorizer-identity-source.ts

```diff
@@ -0,0 +1,41 @@
+import * as lambda from '@aws-cdk/aws-lambda';
```

Comment:
There doesn't seem much of a difference between this test and the `integ.request-authorizer.ts` besides specifying the identity sources. 

Usually, separate integration tests are written when either there are significantly different ways to set something up or there is two major code paths that set up the same infrastructure. For property differences, that result in Cfn property differences, unit tests are sufficient.

Can we call this one `integ.request-authorizer.ts` and drop the other?